schema/method_units/mathematical/opt/ndiff/direct.json

```diff
@@ -0,0 +1,20 @@
+{
+    "schemaId": "method-unit-opt-ndiff-direct",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "Direct algorithms for the optmization of non-differentiable functions",
```

Comment:
here and below too :)

Reply:
🙈 seems like I have to optimize my spelling. (Ok, I'll show myself out ^^)

Reply:
<img src="https://media2.giphy.com/media/l0K4mbH4lKBhAPFU4/giphy.gif"/>